### ISSUE TITLE
Removal of my email

### ISSUE THREAD

**Issue by miclovich** [unicefuganda/messenger#1]
Hello team, 
over the past 2 days I've been spammed by messenger, and I can't read my emails (which may be time-sensitive and urgent), please, can one of you remove my mail from the `localsettings.py` that messenger is using?

This will be highly appreciated.

**Comment by miclovich** [comment]
In the future, can we follow the guidelines listed here

http://support.google.com/a/bin/answer.py?hl=en&answer=1366776

Log files/errors should be send to some google group, and periodically one of you can check that. I think you can all agree with me that there are sensitive emails that one of you could receive that could mean a lot but because of the receive quota that Google gives gmail users, you may not receive it.

**Comment by raybesiga** [comment]
Totally agree with this.

On Wed, Nov 28, 2012 at 12:47 PM, Victor Miclovich <notifications@github.com

> wrote:
> 
> In the future, can we follow the guidelines listed here
> 
> http://support.google.com/a/bin/answer.py?hl=en&answer=1366776
> 
> Log files/errors should be send to some google group, and periodically one
> of you can check that. I think you can all agree with me that there are
> sensitive emails that one of you could receive that could mean a lot but
> because of the receive quota that Google gives gmail users, you may not
> receive it.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/unicefuganda/messenger/issues/1#issuecomment-10796388.

## 

With optimism,

Raymond Besiga, Web Developer
h: http://raybesiga.com
w: http://wurknow.com
b: http://ipatterns.net
m: +256-791-160903

Stalk me: http://twitter.com/raybesiga

**Comment by asseym** [comment]
I agree with this approach. My account did not receive any email for like
24 hours after this incident. And these days messenger keeps going down
every now and then... wonder why? Moses? Sam? Raymond? whats happening?

2012/11/28 Victor Miclovich notifications@github.com

**Comment by sekiskylink** [comment]
Hi Alfred,
We're sorry for the inconviniences this caused you. You know our messenger
app runs in a while loop trying to connect to all databases so if it fails
to connect to any DB it throws up an error that is then sent to our emails.
This happened when we upgraded postgresql to 9.1 on production. The
challenge is that we did not turn off messenger in between.
We have currently disabled you in the localsettings.

Regards,
Sam
